rma_sale_mrp/models/account_move.py

```diff
@@ -13,11 +13,12 @@ def _check_rma_invoice_lines_qty(self):
             "Product Unit of Measure"
         )
         lines = super()._check_rma_invoice_lines_qty()
-        if lines:
-            return lines.sudo().filtered(
-                lambda r: (
-                    r.rma_id.phantom_bom_product
-                    and float_compare(r.quantity, r.rma_id.kit_qty, precision) < 0
-                )
+        if not lines:
```

Comment:
Please just add the new condition instead of changing code adding an extra return. It's not adding value and makes the code more confusing.

Reply:
done